### PR TITLE
Add Kentico Kontent to list of topics

### PR DESCRIPTION
**Pull request by ileshmistry**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Adding a widely known CaaS CMS

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/kentico-kontent/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+created_by: Ilesh Mistry
+display_name: Kentico Kontent
+github_url: https://github.com/kentico
+logo: kontent.png
+related: cms, content-delivery
+released: 2016
+short_description: Kentico Kontent provides environment for easy content collaboration and features flexible API to power modern websites and apps.
```
Comment (kenyonj) [inline topics/kentico-kontent/index.md]: Thoughts on this?
```suggestion
short_description: Kentico Kontent provides an environment for easy content collaboration and features a flexible API to power modern websites and apps.
```

File: topics/kentico-kontent/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+topic: kentico-kontent
+url: https://kontent.ai
+---
+Kentico Kontent is a Content as a Service system that replaces legacy CMS. It offers unified environment for content collaboration, flexible APIs for content management and delivery, content delivery network and SDKs for building websites and apps with the modern development stack.
```
Comment (kenyonj) [inline topics/kentico-kontent/index.md]: Thoughts on this?
```suggestion
Kentico Kontent is a Content as a Service system that can replace a legacy CMS. It offers a unified environment for content collaboration, flexible APIs for content management and delivery, and content delivery network and SDKs for building websites and apps with a modern development stack.
```

File: topics/kentico-kontent/index.md (hunk 1 above)
Comment (petrsvihlik) [reply topics/kentico-kontent/index.md]: lgtm

File: topics/kentico-kontent/index.md (hunk 2 above)
Comment (petrsvihlik) [reply topics/kentico-kontent/index.md]: lgtm

Comment (ileshmistry) [reply topics/kentico-kontent/index.md]: @kenyonj I've made the changes and hopefully it's okay now?

**Comment by kenyonj** [comment]
Looks like some of the suggestions didn't go through, can you take another look?

**Comment by ileshmistry** [comment]
@kenyonj okay, thanks, I've updated the index file again, hopefully, it will come through now?

**Comment by ileshmistry** [comment]
@kenyonj Hi, do you know when it will appear on the Topics page?

**Comment by kenyonj** [comment]
@ileshmistry it's already live here: https://github.com/topics/kentico-kontent. Featured topics on the github.com/topics page are controlled internally.

**Comment by petrsvihlik** [comment]
@ileshmistry @kenyonj would it make sense to also update the description of the [`kentico-cloud`](https://github.com/topics/kentico-cloud) topic to indicate that it's obsolete and point people to `kentico-kontent` instead?

**Comment by kenyonj** [comment]
@petrsvihlik Feel free to open a PR on the topic and add `kentico-cloud` as an alias.

**Comment by ileshmistry** [comment]
@petrsvihlik @kenyonj I can do this if you want me to?

How do we add an alias?

**Comment by kenyonj** [comment]
See here for an example: https://github.com/github/explore/blob/master/topics/3d/index.md
It's a comma separated list

**Comment by ileshmistry** [comment]
@petrsvihlik @kenyonj Thanks, I've created a new PR for this - https://github.com/github/explore/pull/1144

**Pull request merged**